client/my-sites/stats/stats-module-utm/useMockData.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useState } from 'react';
+
+// TODO: Replace mock data with real data.
+// The mock data is behind a network request to simulate a real-world scenario.
+
+const mockUTMData = [
+	{
+		source: 'google',
+		medium: 'cpc',
+		label: 'google / cpc',
+		value: 100,
+		children: [
+			{ label: 'hello world', value: 50 },
+			{ label: 'another world', value: 25 },
+		],
+	},
+	{
+		source: 'bing',
+		medium: 'organic',
+		label: 'bing / organic',
+		value: 50,
+	},
+	{
+		source: 'yahoo',
+		medium: 'organic',
+		label: 'yahoo / organic',
+		value: 25,
+	},
+	{
+		source: 'facebook',
+		medium: 'social',
+		label: 'facebook / social',
+		value: 10,
+	},
+	{
+		source: 'twitter',
+		medium: 'social',
+		label: 'twitter / social',
+		value: 5,
+	},
+];
+
+async function fetchMockDataAsync() {
+	// Make an API request. This will fail without an API key but we don't care.
+	const response = await fetch( `https://api.api-ninjas.com/v1/loremipsum?paragraphs=4` );
```

Comment:
maybe we could skip the data fetch and only stick with the mock data here? It doesn't feel right to have it in the code even it's behind feature flag. What do you think?

Reply:
No strong feelings on this from me. I'll pull it out and update the branch.

Reply:
Removed the API call in favour of a short timeout to simulate a network request.